tests/testthat/test-fb_filter_species_by_traits_coverage.R

```diff
@@ -0,0 +1,114 @@
+# Fake datasets ----
```

Comment:
Same comments as above! You added all these species_traits datasets, now we have to implement tests for them!

Reply:
Done in 2f4ed9c